src/herbivoresAndCarnivores.test.js

```diff
@@ -124,4 +121,36 @@ describe('Animal', () => {
     expect(Animal.alive.includes(tiger))
       .toBe(true);
   });
+
+  it(`Herbivores should be removed from 'Animal.alive' array, when their health is 0 (but Zebra is not the last defined animal)`, () => {
+    const zebra = new Herbivore('Zebra');
+    const tiger = new Carnivore('Tiger');
+
+    tiger.bite(zebra);
+    tiger.bite(zebra);
```

Comment:
This test is similar to the previous one and checks if the Herbivore is removed from the 'Animal.alive' array when its health reaches 0. Make sure the removal logic is correctly implemented.